mne/preprocessing/nirs/_tddr.py

```diff
@@ -0,0 +1,143 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#          Frank Fishburn
+#
+# License: BSD (3-clause)
+
+
+import numpy as np
+
+from ... import pick_types
+from ...io import BaseRaw
+from ...utils import _validate_type
+
+
+def TDDR(raw):
+    r"""Apply temporal derivative distribution repair to data.
+
+    Applies temporal derivative distribution repair
+    :footcite:`fishburn2019temporal`. This approach removes baseline shift
+    and spike artifacts without the need for any user-supplied parameters.
```

Comment:
function names should not have any capital letter. Also I would prefer not to have any acronym as I find it hard to understand for new comers. Acronyms have often different meaning across communities

Reply:
Thanks for the feedback. I had been internally debating this.